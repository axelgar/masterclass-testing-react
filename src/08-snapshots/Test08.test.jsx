// import { http, HttpResponse } from "msw";
// import { setupServer } from "msw/node";

// const server = setupServer(
//   http.get("/greeting", () => {
//     return HttpResponse.json({ greeting: "hello there" });
//   }),
// );

// beforeAll(() => server.listen());
// afterEach(() => server.resetHandlers());
// afterAll(() => server.close());
