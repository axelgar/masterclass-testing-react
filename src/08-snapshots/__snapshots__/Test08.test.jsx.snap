// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test08 > snapshot 1`] = `
<body>
  <div>
    <section
      class="flex flex-col gap-4"
    >
      <h2
        class="font-semibold"
      >
        08 - Snapshots
      </h2>
      <div
        class="border border-gray-500 p-2 text-center"
      >
        <button
          class="group/wishlist inline-flex max-w-xs cursor-pointer items-center justify-center gap-2 rounded-sm bg-red-400 px-3 py-1 font-semibold text-white disabled:cursor-not-allowed disabled:bg-gray-400 disabled:text-gray-200"
        >
          <svg
            aria-hidden="true"
            class="h-5 w-5 text-white group-disabled/wishlist:text-gray-200"
            data-slot="icon"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M21 8.25c0-2.485-2.099-4.5-4.688-4.5-1.935 0-3.597 1.126-4.312 2.733-.715-1.607-2.377-2.733-4.313-2.733C5.1 3.75 3 5.765 3 8.25c0 7.22 9 12 9 12s9-4.78 9-12Z"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
          <span
            class="truncate block max-w-full"
          >
            Add to wishlist
          </span>
        </button>
      </div>
    </section>
  </div>
</body>
`;
